File: test-fixtures/typescript/interface.ts
// Test fixture for TypeScript interface symbol navigation
// Usage: Place cursor on 'Person' in line 10 and press F12
// Expected: Cursor jumps to line 4 (interface definition)

interface Person {
  name: string;
  age: number;
}

const person: Person = {
  name: 'Bob',
  age: 25
};

console.log(person.name);

File: test-fixtures/go/interface.go
// Test fixture for Go interface symbol navigation
// Usage: Place cursor on 'Greeter' in line 22 and press F12
// Expected: Cursor jumps to line 9 (interface definition)

package main

import "fmt"

type Greeter interface {
	Greet() string
}

type Person struct {
	Name string
}

func (p Person) Greet() string {
	return "Hello, " + p.Name
}

func main() {
	var g Greeter = Person{Name: "Charlie"}
	fmt.Println(g.Greet())
}
